tests/flow/test_config.py

```
@@ -325,3 +326,90 @@ def test11_set_get_node_creation_buffer(self):
+        # try setting TEMP_FOLDER to a file
+        # expecting config update to fail
+        fd, file_path = tempfile.mkstemp()
+        os.close(fd)
+
+        # try updating TEMP_FOLDER
+        try:
+            self.set_temp_folder(file_path)
+            # setting TEMP_FOLDER to a file should have failed
+            self.env.assertFalse(True)
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)